src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespacearound/InputWhitespaceAroundAllowEmptyCompactCtors.java

```diff
@@ -63,6 +63,18 @@ record MyRecord6() {
     }
 
     record TestRecord7() {
-        public TestRecord7 {} // ok
+        public TestRecord7{} // violation, no WS before '{'
     }
+
+    class foo {
+        public foo() {}
+    }
+
+    class foo2 {
+        public foo2(){} // violation, no WS before '{'
+    }
+    class foo3 {
+            public foo3(){System.out.println();} // violation, no WS before '{'
+                                                 // and '}' and after '{'
```

Comment:
Please change comment to make it more clear

Reply:
done